Jenkinsfile

```diff
@@ -13,6 +13,10 @@ def addStages() {
       codenarc: {
         codenarc.check('**/Jenkinsfile,**/*.gradle,**/*.groovy')
       },
+      junit: {
+        sh './gradlew test'
+        junit 'build/test-results/**/*.xml'
```

Comment:
where does `junit` come from?

Reply:
The JUnit Jenkins plugin, which is bundled with Jenkins... or maybe it's not bundled, but in that case it's a dependency of some other plugin we use. Either way, it's automatically installed whenever we deploy a new Jenkins.